package/src/components/Attachment/__tests__/Attachment.test.js

```diff
@@ -148,15 +148,15 @@ describe('Attachment', () => {
 
     jest.spyOn(Linking, 'openURL').mockImplementation(jest.fn().mockResolvedValue(true));
 
-    const { result, waitForNextUpdate } = await renderHook(() => useGoToURL());
-    const [error, openURL] = result.current;
-
-    act(openURL);
+    const { result } = renderHook(() => useGoToURL());
+    const [, openURL] = result.current;
+    await act(openURL);
+    const [error] = result.current;
 
-    await waitForNextUpdate(() => {
+    await waitFor(() => {
       expect(Linking.canOpenURL).toHaveBeenCalled();
       expect(error).toBeTruthy();
-      expect(Linking.openURL).not.toHaveBeenCalled();
+      expect(Linking.openURL).toHaveBeenCalled();
```

Comment:
This test was written wrongly and it didn't really execute any of those `expect` blocks. It has been fixed now